docs/releases/v1.1.0.rst

```diff
@@ -25,6 +25,9 @@
     -[fix] Fixed a crash which could have occurred when sorting very long
         identical or nearly identical strings. [#3134]
 
+    -[fix] Fixed a crash when :attr:`.ltypes` or :attr:`.stypes` properties
+        were accessed for a frame containing array columns. [#3142]
```

Comment:
This is not fixing something from the previous  version, but rather something that we just added recently. As such, it's not needed to be mentioned in the release notes.

Reply:
Yeah, let's remove it.